presidio-anonymizer/tests/test_anonymizer.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from anonymizer import app
+
+
+class TestAnonymizer(unittest.TestCase):
```

Comment:
What are we planning to test here? The controller layer?

Reply:
This test for now, is just a POC to see the method is working with flask and REST. It will change once we write code here.